src/components/CrimCodeGridList.js

```diff
@@ -9,31 +9,28 @@ import Icon from 'react-native-vector-icons/MaterialIcons';
 //USER IMPORTS
 import styles, {colors} from '../assets/styles';
 
-const CrimCodeGridList = props => {
+export const CrimCodePartsList = props => {
```

Comment:
Why can't this component be genericized to be used for both PartsCCScreen and SectionCCScreen? 

These 2 are the exact same code except for a few different prop names; make the key names the same and pass them into the component you've created.

Reply:
I could do that.. but the styling is slightly different.  But I will proceed as you suggest.